README.md

```diff
@@ -4,4 +4,45 @@ Configuration for creating a NGINX Unit package for Arch Linux
 
 Based on [the aur nginx-unitd](https://aur.archlinux.org/packages/nginx-unitd) package; updated to build 1.29.0 instead of 1.27.0.
 
-Some day I will make time to turn this into a real boy. But it is not this day.
+
+# Requirements
+
+If you are running this locally in Manjaro / Arch use the following to ensure all required libraries are installed:
+
+```
+pacman --noconfirm -Syu base-devel go ruby php-embed nodejs npm
+```
+
+# Usage
```

Comment:
Should probably have this part filled out before submitting to unit (sorry if this was already on your list).  I would also suggest a list of the various args provided to the container and their roles.